### PR TITLE
Update Suggested Privacy Policy text to use the non-deprecated class.

### PR DESCRIPTION
**Pull request by garretthyder**
## All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Update Suggested Privacy Policy text to utilize `privacy-policy-tutorial` css class instead of `wp-policy-help` as it was deprecated. Also wrap the contents in the `wp-suggested-text` div to style the section to match WordPress. This follows from [WPCoreTrac#49282](https://core.trac.wordpress.org/ticket/49282) and although back-compat is being introduced in WP5.4 as of [WPChangeset#47112](https://core.trac.wordpress.org/changeset/47112) this change will better support users of WP5.1-5.4

### How to test the changes in this Pull Request:

1. Install updated WooCommerce on WP between 5.1 and 5.3
2. Navigate to the Privacy Policy Guide in WordPress (`/wp-admin/privacy-policy-guide.php`)
3. Confirm the styling of the guide and the copy action only grabs the appropriate contents.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

Updated Privacy Policy Guide Suggested text to support the `privacy-policy-tutorial` and `wp-suggested-text` classes added in WP5.1 and deprecates use of `wp-policy-help` class.

### CODE REVIEW

**Comment by garretthyder** [comment]
Note: This also removes the wrapping `<div contenteditable="false">` as it's now unnecessary. As long as the content uses the proper `privacy-policy-tutorial` class then the WP Privacy Policy Guide section copy action will avoid copying that content. The contenteditable divs were there originally when the contents was fully copied due to not using the appropriate class, and when originally the entire guide would be dumped into the WYSIWYG editor with `wp-policy-help` sections being highlighted.

**Comment by claudiosanches** [comment]
We can't support it yet, since we still need to provide support for WP 5.0.

**Comment by garretthyder** [comment]
Hi @claudiosanches thanks for the review.

TLDR; You're actually safe the .wp-policy-help version was committed but was replaced before release of 4.9.6.

I provided some misinformation, sorry my eyes lied to me. The [coreTrac#43980](https://core.trac.wordpress.org/ticket/43980) had a version of 5.1... but the milestone it was committed to was 4.9.6 the inception of privacy tools in WP....
So the version you were utilizing came from a beta version of 4.9.6 that never saw the light of day.

You can confirm this by viewing the privacy policy content on the 4.9.6 tag;
https://github.com/WordPress/wordpress-develop/blob/4.9.6/src/wp-admin/includes/misc.php
*Just to be safe I even installed 4.9.6 to confirm the classes were .privacy-policy-tutorial and not .wp-policy-help which apparently was introduced in [coreTrac#43473](https://core.trac.wordpress.org/ticket/43473#no0)

Please confirm and I think that means we're safe to merge.

**Comment by Konamiman** [comment]
Tested in WP 5.0 and 5.3.2, and in both cases the difference is that `<p>` sections inside the "Source: WooCommerce" section appear in italics after applying the changes; other elements (mostly `<li>`) appear unchanged. Is this the expected behavior?

**Comment by Konamiman** [comment]
I see that the text copied to the clipboard changes too, so the "meta" explanations aren't included in the copied text. I guess that's on purpose too.

**Review by Konamiman — APPROVED** [review]

**Comment by claudiosanches** [comment]
Failing e2e tests, but tests related to variations on the front-end, so not a problem on this PR.

**Comment by garretthyder** [comment]
Thanks for reviewing @Konamiman 

Good catch with the non-p tags. As the core privacy policy guide didn't hold any they weren't included in the core css implementation. I've opened [core#49772](https://core.trac.wordpress.org/ticket/49772) to address this moving forward and support additional elements beyond the <p> tags.

I can add some css to this PR to style <ul> in the same italics for previous versions than 5.4 and now that 5.4 is released there's a bit more styling due to [core#44621](https://core.trac.wordpress.org/ticket/44621) which can also be accounted for until a core fix is introduced.
Screen Shot 2020-04-01 at 1 18 08 PM
See above for new styling.

Here is some version compatible CSS to address <ul>'s in a consistent manner (this is what I propose adding to the PR);
```
.branch-4-9 .wp-suggested-text ul,
.branch-4-9 .wp-suggested-text ol,
.branch-5-0 .wp-suggested-text ul,
.branch-5-0 .wp-suggested-text ol,
.branch-5-1 .wp-suggested-text ul,
.branch-5-1 .wp-suggested-text ol,
.branch-5-2 .wp-suggested-text ul,
.branch-5-2 .wp-suggested-text ol,
.branch-5-3 .wp-suggested-text ul,
.branch-5-3 .wp-suggested-text ol {
    font-style: italic;
}
.branch-5-4 .policy-text ul:not(.privacy-policy-tutorial):not(.wp-policy-help),
.branch-5-4 .policy-text ol:not(.privacy-policy-tutorial):not(.wp-policy-help) {
	background-color: #fff;
	margin: 0;
	padding: 1em;
}
.branch-5-4 .hide-privacy-policy-tutorial ul:not(.privacy-policy-tutorial):not(.wp-policy-help),
.branch-5-4 .hide-privacy-policy-tutorial ol:not(.privacy-policy-tutorial):not(.wp-policy-help) {
	margin: 1em 0;
	padding: 0;
}
```

As to the question concerning the copy to clipboard, yes you're correct only the headings and contents should be copied and not the tutorial text (help text). This is done by hiding it via css on the .privacy-policy-tutorial (and for back-compat now the .wp-policy-help) found here;
https://github.com/WordPress/wordpress-develop/blob/7ede058b163b3620cddad3b816bd3a5bfd59f135/src/wp-admin/css/edit.css#L750-L753

Let me know if I should include the above css shim and happy to answer any other questions. Also 'Suggested Text:' is a prefix WP core uses I can add that as well, see screenshot above.

P.S. @claudiosanches is there something I should change to resolve the e2e tests?

**Comment by claudiosanches** [comment]
> @claudiosanches is there something I should change to resolve the e2e tests?

Those are already fixed on master, so only if you want to rebase, but there's no problem.

And please include those new CSS you suggested, it will be great!
Thanks!

**Comment by garretthyder** [comment]
Thanks @claudiosanches I've added a privacy.scss file for the css back-compat shim there. I modified it a bit through testing. Would appreciate some double-checks.

The new styling in 5.4 to accommodate WC UL Lists;
Screen Shot 2020-04-01 at 4 34 51 PM

One last question, do you care for me to add the 'Suggested text:' prefix that's used in the default WP privacy policy content? See below for sample of this;
Screen Shot 2020-04-01 at 4 35 54 PM

**Comment by Konamiman** [comment]
This is what I see now when testing in WP 5.0:

image

**Comment by garretthyder** [comment]
Seems there's some [git issues](https://www.githubstatus.com/) today so I'll let those settle down and if the commits don't appear will start a new PR.
Screen Shot 2020-04-02 at 1 45 27 PM

**Comment by garretthyder** [comment]
Doesn't seem they'll refire webhooks so going to move this to a new PR and fix the connection with the branch.

**Comment by garretthyder** [comment]
Apologies @Konamiman & @claudiosanches I've migrated this to #26072 due to github issues today. It contains a fix for the 5.0 issues flagged.
